components/Bookmarks/BookmarkMiniFeed.js

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useRef } from 'react'
+import { compose, graphql } from 'react-apollo'
+import { css } from 'glamor'
+import {
+  colors,
+  fontStyles,
+  mediaQueries,
+  Loader,
+  linkRule
+} from '@project-r/styleguide'
+
+import Bookmark from '../ActionBar/Bookmark'
+import UserProgress from '../ActionBar/UserProgress'
+import withT from '../../lib/withT'
+import { withMembership } from '../Auth/checkRoles'
+import Link from '../Link/Path'
+
+import { getBookmarkedDocuments } from './queries'
+
+const BookmarkMiniFeed = ({ t, data, ...props }) => {
+  return (
+    <Loader
+      loading={data.loading}
+      error={data.error}
+      render={() => {
+        const nodes = data.me.collection.items.nodes
+        return (
+          <div {...styles.tilesContainer} {...props}>
+            {nodes.slice(0, 8).map(node => {
```

Comment:
`node.document` can be null (e.g. staging, doc not available or unpublished). Let's just filter those out:
```suggestion
            {nodes.filter(node => node.document).slice(0, 8).map(node => {
```